src/app/ApplicationTemplate.Access/ApiClients/MinimumVersion/MinimumVersionRepositoryMock.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Reactive.Subjects;
+
+namespace ApplicationTemplate.DataAccess;
+
+/// <summary>
+/// A mock implementation of the minimum version repository. Used for testing.
+/// </summary>
+public sealed class MinimumVersionRepositoryMock : IMinimumVersionReposiory
```

Comment:
This should probably implement `IDisposable`? Even if this is a singleton and lives until the application is killed, it would make it cleaner if someone wanted to optimize his application and only use this one time on start and then dispose it.

See https://github.com/nventive/UnoApplicationTemplate/blob/375ef8b6c72b340f793c361e0fcb6703cd63010d/src/app/ApplicationTemplate.Shared.Views/Framework/Connectivity/ConnectivityProvider.cs#L66 for more details.

Reply:
This is a mock service thought? I think it would make more sense for someone to add it later to their own implementation if they want this.

Reply:
Sure but I feel like it's a good coding practice to always make it available right away. I don't think anyone should have to think about it and change it, it should be available as soon as we have stuff to dispose.